Updates vu projects

xulman opened the pull request:
Updated the main table with repositories to reflect the recent shuffle around Tomancak lab -- where old projects and Vlado's-experimental stuff were moved out. This PR reflects on these changes.

btw, thanks @tinevez  for archiving the first generations of the sciview and collaborative projects for me.

xulman committed:
updated Distribution column for some VU's projects

xulman committed:
cosmetics on the formatting of the table (no outside visible change)

xulman commented:
I also added a commit that just "beautifies" the table in `.md` .

tinevez commented:
Thanks!
I thought you wanted to remove the archived projects. Do you want them to stay here?

xulman commented:
aah, true, maybe we should remove them from the list

or, mark them *legacy*, and shift at the very end of the list... apparently it's hard to let them go :-)


you're the boss, please decide how you like the main page to appear

tinevez commented:
I would suggest putting on the frontpage only the artifacts that are shipped and ready to be used. There is now a lot of artifacts in the org, and the frontpage must play a role where it facilitates visitors grasping the main ones first.

xulman committed:
README.md, removed 'mastodon-collaborative' from the main table.

xulman committed:
README.md, 'mastodon-sciview' replaced with '...sciview (v2)', added credit to Samuel

xulman commented:
two new commits added just above:
- removed the archived repo about the old collaborative tracking
- replaced the old mastodon-sciview bridge with the new one(*)

(*) here Samuel (from Ulrik) has done a lot of good work and he's basically now in charge of that repo... I couldn't have left just "VU" in the maintainer column, he should be there too... so I have added him... not sure you like it that way, please suggest

(*) I would not remove this mastodon-sciview entry, especially now when J21 became reality in Fiji... now sciview can be in Fiji too and with it this bridge can make its way on some update site; and we could update the "obtain from repo" information... yes, it is basically similar to yours OpenGL renderer and could feel now like redundant... delicate now
